without this the below error messages were coming continuously

TWL4030 module irq 369 is disabled but can't be masked!
<2>TWL4030 module irq 369 is disabled but can't be masked!

Signed-off-by: Arun C <arunedarath@mistralsolutions.com>
---
 arch/arm/mach-omap2/board-omap3evm.c |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/arch/arm/mach-omap2/board-omap3evm.c
b/arch/arm/mach-omap2/board-omap3evm.c
index 92f70ce..f9fd06b 100644
--- a/arch/arm/mach-omap2/board-omap3evm.c
+++ b/arch/arm/mach-omap2/board-omap3evm.c
@@ -22,6 +22,7 @@
 #include <linux/input.h>
 #include <linux/spi/spi.h>
 #include <linux/spi/ads7846.h>
+#include <linux/i2c/twl4030.h>

 #include <mach/hardware.h>
 #include <asm/mach-types.h>
@@ -176,6 +177,7 @@ static struct omap_kp_platform_data omap3evm_kp_data = {
 	.keymap 	= omap3evm_keymap,
 	.keymapsize	= ARRAY_SIZE(omap3evm_keymap),
 	.rep		= 1,
+	.irq		= TWL4030_MODIRQ_KEYPAD,
 };

 static struct platform_device omap3evm_kp_device = {

